integreat_cms/cms/models/pages/page_translation.py

```diff
@@ -100,8 +100,37 @@ def short_url(self):
             "public:expand_page_translation_id", kwargs={"short_url_id": self.id}
         )
 
+    @cached_property
+    def mirrored_translation_text_internal(self):
```

Comment:
```suggestion
    def mirrored_translation_text(self):
```
I think we can combine these methods into one.